test/integration/workarea/storefront/affirm_integration_test.rb

```diff
@@ -0,0 +1,117 @@
+require 'test_helper'
+
+module Workarea
+  module Storefront
+    class Affirm < Workarea::IntegrationTest
+      setup do
```

Comment:
Use a method and `setup :method_name` so people can decorate the setup if they need to.